app/upgrade_0_11_4.go

```diff
@@ -0,0 +1,387 @@
+package app
+
+import (
+	"bufio"
+	"bytes"
+	_ "embed"
+	"encoding/binary"
+	"encoding/csv"
+	"encoding/hex"
+	"fmt"
+	wasmTypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	"github.com/cosmos/cosmos-sdk/types"
+	"strings"
+)
+
+//go:embed reconciliation_data.csv
+var reconciliationData []byte
+
+//go:embed reconciliation_data_testnet.csv
+var reconciliationDataTestnet []byte
+
+var reconciliationBalancesKey = prefixStringWithLength("balances")
+
+var NetworkInfos = map[string]NetworkConfig{
+	"fetchhub-4": {
+		ReconciliationInfo: &ReconciliationInfo{
+			TargetAddress:   "fetch1tynmzk68pq6kzawqffrqdhquq475gw9ccmlf9gk24mxjjy6ugl3q70aeyd",
+			InputCSVRecords: readInputReconciliationData(reconciliationData),
+		},
+		Contracts: &ContractSet{
+			Reconciliation: &Reconciliation{
+				Addr:     "fetch1tynmzk68pq6kzawqffrqdhquq475gw9ccmlf9gk24mxjjy6ugl3q70aeyd",
+				NewAdmin: getStringPtr("fetch15p3rl5aavw9rtu86tna5lgxfkz67zzr6ed4yhw"),
+				NewLabel: getStringPtr("reconciliation-contract"),
```

Comment:
We need to implement change for label value - it is present hrere in the config, but not implemented.